examples/nl-core-StrongKidsScore-01.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-StrongKidsScore-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-StrongKidsScore"/>
+   </meta>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="108341000146107"/>
+         <display value="StrongKids-screeningsinstrument"/>
```

Comment:
This should be with capital letters like "STRONGkids-screeningsinstrument"

Reply:
Wordt niet gewijzigd in zib (in 2020-release)